src/nodejs/unit-http/http_server.js

```diff
@@ -11,10 +11,13 @@ const util = require('util');
 const unit_lib = require('./build/Release/unit-http');
 const Socket = require('./socket');
 const WebSocketFrame = require('./websocket_frame');
-
+const stream = require('stream');
+const Writable = stream.Writable;
+const Readable = stream.Readable;
 
 function ServerResponse(req) {
     EventEmitter.call(this);
+    Writable.call(this);
```

Comment:
Writable already inherits Stream, which, in turns, inherits EventEmitter. Is it still required to inherit EventEmitter explicitly?

Reply:
@mar0x inherit EventEmitter is useless, stream.Writable is already inherited